schema.sql

```diff
@@ -73,4 +73,10 @@ CREATE TABLE visits (
 
 );
 
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
 
+ALTER TABLE owners ALTER COLUMN age DROP NOT NULL;
+
+CREATE INDEX animal_id on visits (animal_id ASC);
+CREATE INDEX vet_id_asc on visits (vet_id ASC);
+CREATE INDEX email_id on owners(email ASC); 
```

Comment:
- [ ] The first pair of screenshots don't show the decrease of execution time
![Screenshot from 2021-12-06 16-41-09](https://user-images.githubusercontent.com/56886064/144865788-66ccc73c-004c-4f0c-b3b6-efcc99b11538.png)
=> Would you address this issue, because we are expecting to see the execution time goes down because of the fixes we've made from our schema.sql file

Reply:
Hello @redwing555 Thank you for the detailed explanations and for re-arranging the screenshots for better context, you rock :rocket: